tests/30rooms/06invite.pl

```diff
@@ -107,3 +111,424 @@
          Future->done(1);
       });
    };
+
+test "Can invite existing 3pid",
+   requires => [qw( user more_users test_http_server_hostandport await_http_request )],
```

Comment:
In fact this is now done; see 80f41272167e4f1f5c3e6bde38953b2ee8d61c8c

This commit also includes examples of the changes you'll need here to adapt for it. Should be relatively simple.